kitsune/sumo/static/sumo/js/codemirror.sumo-hint.js

```diff
@@ -0,0 +1,38 @@
+(function(mod) {
+  if (typeof exports == "object" && typeof module == "object") // CommonJS
+    mod(require("../../lib/codemirror"));
+  else if (typeof define == "function" && define.amd) // AMD
+    define(["../../lib/codemirror"], mod);
+  else // Plain browser env
+    mod(CodeMirror);
+})(function(CodeMirror) {
+  "use strict";
+
+  var WORD = /[\w$-]+/, RANGE = 500;
+
+  CodeMirror.registerHelper("hint", "sumo", function(editor, options) {
+    var word = options && options.word || WORD;
+    var range = options && options.range || RANGE;
```

Comment:
You could rename this file to `codemirror.sumo-hints.js.es6` and use es6 default parameters like this:

```js
CodeMirror.registerHelper("hint", "sumo", function(editor, {word=/[\w$-]+/, range=500}) {
```